packages/experiments/src/components/Microfeedback/examples/Microfeedback.Basic.Example.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Microfeedback } from '../Microfeedback';
+import { IMicrofeedbackQuestion } from '../Microfeedback.types';
+
+export class MicrofeedbackBasicExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const followUpOnThumbsDown: IMicrofeedbackQuestion = {
+      options: ['Translation is incorrect', 'Context is incorrect', 'Language can be better'],
+      question: 'Please help us improve'
+    };
+
+    const followUpOnThumbsUp: IMicrofeedbackQuestion = {
+      options: ['Translation is great', 'Context is great'],
+      question: 'Please help us improve'
+    };
+
+    return (
+      <div>
+        <Microfeedback
+          ThumbsDownQuestion={followUpOnThumbsDown}
+          ThumbsUpQuestion={followUpOnThumbsUp}
+          thumbsUpTitle="Like"
+          thumbsDownTitle="Dislike"
```

Comment:
👍 I think using `sendFeedback` and `sendFollowupItem` would be useful here, even if they just show a popup or log to console.

Reply:
Fixed